test/manual/package.json

```diff
@@ -11,6 +11,6 @@
   },
   "devDependencies": {
     "@types/node": "^14.14.22",
-    "typescript": "^4.1.3"
+    "typescript": "4.6.3"
```

Comment:
Why not go higher?

Reply:
Wanted to break this up in pieces a bit, but I guess there's not reason not to bump this to 5 :)